### PR TITLE
ProfilePicture useRef component

### PR DESCRIPTION
**Pull request by esthereis**

### CODE REVIEW

File: src/components/sections/ProfileImage.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+    try {
+      const imgUrl = await fileToDataString(file);
+      setSelectedImage(imgUrl);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  function handleRefClick() {
```
Comment (artursoareshsv) [inline src/components/sections/ProfileImage.tsx:35]: I think you could remove this function and implement directly in the onClick.

File: src/components/sections/ProfileImage.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        onChange={(e) => {
+          handleFileChange(e);
+        }}
+        ref={ref}
+      />
+      <img
+        src={selectedImage}
+        alt="profile-image"
+        className="profile-image"
+        onClick={() => handleRefClick()}
```
Comment (artursoareshsv) [inline src/components/sections/ProfileImage.tsx:55]:
```suggestion
        onClick={() => ref.current.click()}
```

File: src/components/sections/ProfileImage.tsx (hunk 1 above)
Comment (artursoareshsv) [reply src/components/sections/ProfileImage.tsx:35]: Have a look at my suggestion bellow.

File: src/components/sections/ProfileImage.tsx (hunk 2 above)
Comment (esthereis) [reply src/components/sections/ProfileImage.tsx:55]: Makes sense! I'll do it.

**Pull request merged**